src/dvc_data/hashfile/meta.py

```diff
@@ -79,12 +80,32 @@ def from_dict(cls, d: Dict[str, Any]) -> "Meta":
         return cls(**kwargs)
 
     def to_dict(self) -> Dict[str, Any]:
```

Comment:
Also, we can write it as follows if we don't want to duplicate.

```python
    def to_dict(self):
        ret = {}

        for field in self.fields:
            if field in (self.PARAM_INODE, self.PARAM_MTIME):
                continue
            if value := getattr(self, field):
                ret[field] = value
        return ret
```

This is twice as fast as asdict, but 4x slower than handwritten converter
(~2us for asdict, ~1us for this, and ~250ns for handwritten one).

Reply:
So the code is more complicated and performance is worse? 😄 Seems bad, let's stick with KISS.